ports/esp32/mphalport.c

```diff
@@ -54,6 +44,8 @@
 #include "mphalport.h"
 #include "usb.h"
 #include "usb_serial_jtag.h"
+#include "uart.h"
+#include "driver/uart.h"
```

Comment:
Does it need `driver/uart.h`?

Reply:
Ah probably not anymore, since wrapping the uart write function